sdks/python/src/opik/config.py

```diff
@@ -22,6 +22,8 @@
 
 _SESSION_CACHE_DICT: Dict[str, Any] = {}
 
+MAX_BATCH_SIZE_MB = 50
```

Comment:
As of today we have no mechanism for getting any configuration from the backend. So for now I prefer to keep it like that, since we don't even know how it may look like when we get to adding it.